content/docs/ve/veAdditionalVEC.md

```diff
@@ -7,8 +7,9 @@ date: "2019-05-05T00:00:00+01:00"
 # Prev/next pager order (if `docs_section_pager` enabled in `params.toml`)
 weight: 50
 ---
-## Follow these instructions if you have an existing ExamTools account and have now become accredited with 1 or more additional VECs.
+## Follow these instructions if you have an existing ExamTools account and have now become accredited with one or more additional VECs.
 
+<br />
```

Comment:
no need for a `<br />` tag; Just add a blank line.

Reply:
The <br /> tag is for a second blank line. When creating space, markdown renders multiple carriage returns as a single one (combining them). It's a kludge, sadly.

Reply:
See https://github.com/HamStudy/examtools-docs/pull/15#discussion_r541721316

Reply:
Btw you have to quote your HTML tags even in the markdown here;-)

Reply:
See https://github.com/HamStudy/examtools-docs/pull/15#discussion_r542996260.